src/cil.ml

```diff
@@ -4163,32 +4163,43 @@ class defaultCilPrinterClass : cilPrinter = object (self)
           text "/*" ++ pa ++ text "*/"
       | _ -> pa
     in
+    let printAtomic (a:attributes) =
+      if List.exists (function Attr ("atomic", _) -> true | _ -> false) a then
+        text "_Atomic "
+      else
+        text ""
+    in
     match t with
       TVoid a ->
+        printAtomic a ++
```

Comment:
because `const` and `volatile` are printed in the GCC specific manner, i.e. put into `__attribute__`, but one may not do this for `_Atomic` as GCC will ignore this attribute.

Reply:
But `pAttr` does print `const` and `volatile` without any `__attribute__` just fine, no?

Reply:
you're right!